src/components/TextInput/BaseTextInput/types.ts

```diff
@@ -26,6 +26,9 @@ type CustomBaseTextInputProps = {
     /** Icon to display in right side of text input */
     icon?: IconAsset | null;
 
+    /** Icon to display in left side of text input */
+    iconLeft: IconAsset | null;
```

Comment:
Shouldn't it be marked as optional?
```suggestion
    iconLeft?: IconAsset | null;
```